airflow/providers/common/sql/hooks/sql.py

```diff
@@ -178,6 +179,10 @@ def __init__(self, *args, schema: str | None = None, log_sql: bool = True, **kwa
             "replace_statement_format", "REPLACE INTO {} {} VALUES ({})"
         )
 
+    @property
+    def sa_uri(self) -> URL:
+        raise NotImplementedError
+
```

Comment:
There is no docstring, as well as there is no proper error message. Which might confuse someone who will try to run it against other Hooks